src/Controller/NphBiobankController.php

```diff
@@ -65,6 +66,44 @@ public function participantsAction(Request $request): Response
         ]);
     }
 
+    /**
+     * @Route("/orderlookup", name="nph_biobank_order_lookup")
+     */
+    public function orderLookupAction(
+        Request $request,
+        NphParticipantSummaryService $participantSummary
+    ): Response {
+        $idForm = $this->createForm(OrderLookupIdType::class, null);
+        $idForm->handleRequest($request);
+
+        if ($idForm->isSubmitted() && $idForm->isValid()) {
+            $id = $idForm->get('orderId')->getData();
+
+            $order = $this->em->getRepository(NphOrder::class)->findOneBy([
+                'orderId' => $id
+            ]);
+
+            if ($order) {
+                $participant = $participantSummary->getParticipantById($order->getParticipantId());
+                if (!$participant) {
+                    throw $this->createNotFoundException('Participant not found.');
+                }
+                return $this->redirectToRoute('nph_order_collect', [
```

Comment:
Could you redirect to nph_biobank_order_collect and pass the biobankId instead of participantId as the biobank users won't be able to access the participant id?